runtime/oti/j9nonbuilder.h

```diff
@@ -4219,6 +4221,7 @@ typedef struct J9CRIUCheckpointState {
 	UDATA libCRIUHandle;
 	struct J9VMInitArgs *restoreArgsList;
 	char *restoreArgsChars;
+	U_32 flags;
```

Comment:
I'd suggest putting this at the start of the struct and making it an U_64 so the checks for enabled, etc are cheap if you have a pointer to the struct in hand.